service/awsconfig/v7/key/key.go

```diff
@@ -108,11 +111,18 @@ func ClusterNamespace(customObject v1alpha1.AWSConfig) string {
 	return ClusterID(customObject)
 }
 
+// ClusterOrganization returns the org name from the custom object.
+// It uses ClusterCustomer until this field is renamed in the custom object.
+func ClusterOrganization(customObject v1alpha1.AWSConfig) string {
+	return ClusterCustomer(customObject)
+}
```

Comment:
Adds a new key helper that just calls `ClusterCustomer`. Hopefully the comment is clear and we can fix when we clean up the `awsconfig` CRD.